app/__init__.py

```diff
@@ -35,6 +43,11 @@ def create_app(config_name):
     return application
 
 
+@login_manager.user_loader
+def load_user(user_id):
```

Comment:
Something not 100% clear to me from the docs is when this `load_user` method is called?  Is it only for initial login and cookie setting, or for every page that requires authentication?  If the latter then we'll be calling the auth API for every page load.